src/main/java/org/zalando/nakadi/service/EventStream.java

```diff
@@ -11,13 +12,16 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
 import java.nio.charset.Charset;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Function;
 import java.util.stream.Collectors;
+
```

Comment:
I'm using IntelliJ and the current formatting settings add it and that's why it slips easily in the changes. Do you have an advice how to setup IntelliJ styles properly so that it doesn't mess the imports?